### PR TITLE
Format Locations file

### PR DESCRIPTION
**Pull request by ellisrichardj**
This PR is to add an additional process to format the locations file from the metadata warehouse for use by Augur.  Tab separated values are required, and the first column needs to identify the location type (e.g. CPH or County) - see below  It also adds County locations (from a separate static file) which will allow the display of County-level summaries on the map. 

CPH	01/006/0045	52.0153075045731	-0.414631106731372
CPH	01/012/0152	52.1140624236131	-0.609298037192273
CPH	01/045/0064	52.1872212958073	-0.333624906280236
CPH	01/045/0065	52.2119935594832	-0.356790325017313
CPH	01/075/0012	51.9265023941018	-0.622941104183655
County	Dyfed	52.089816	-4.095032
County	Gwent	51.744687	-2.902416
County	Powys	52.371127	-3.406622

### CODE REVIEW

File: bin/formatLocations.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+def locationfix(location_csv, counties_csv):
+
+    date_out = date.today().strftime('%d%b%y')
+
+    location_df = pd.read_csv(location_csv, dtype='object')
+    location_df.insert(0, 'LocationType', 'CPH')
+    
+    counties_df = pd.read_csv(counties_csv, dtype='object', sep='\t')
+    
+    Frames = [location_df, counties_df]
```
Comment (nick-pestell) [inline bin/formatLocations.py:16]: standard to use lowercase for variables/objects: with the capital F, this makes it look like a class. Same with the Allocations_df below. (I think that might be why github has made them a difference colour ...

File: bin/formatLocations.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    AllLocations_df = pd.concat(Frames, ignore_index=True)
+    AllLocations_df.set_index('LocationType', inplace=True)
+    
+    # write updated locations file
+    AllLocations_df.to_csv('allLocations_{}.tsv'.format(date_out), sep='\t', header=None)
```
Comment (nick-pestell) [inline bin/formatLocations.py:21]: I think you could add `index=False` here and then remove line 18. :)

**Review by nick-pestell — CHANGES_REQUESTED** [review]
Looks good. Just two minor changes to the python file required I think.

File: bin/formatLocations.py (hunk 1 above)
Comment (ellisrichardj) [reply bin/formatLocations.py]: Thanks - no changed

File: bin/formatLocations.py (hunk 2 above)
Comment (ellisrichardj) [reply bin/formatLocations.py]: Good spot - makes sense

**Review by nick-pestell — APPROVED** [review]
Nice. Thanks for making those changes

**Review by Hira-Naveed — COMMENTED** [review]
Good to see the process taken to clean the data!

**Pull request merged**